### PR TITLE
fix: alert input styles

### PR DESCRIPTION
**Pull request by veegandhi**
Alert input fixes

## Description

Fix alert input style which were broken as a result of Popover class name changes.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Screenshots (if appropriate):
alertInput

## Check-list:

- [x] I have read the [Contributing](CONTRIBUTING.md) document.
- [x] I've thought about and labelled my PR/commit message appropriately.
- [x] If this PR introduces breaking changes I've described the impact and migration path for existing applications.
- [x] CI is green (coverage, linting, tests).
- [x] I have updated the documentation accordingly.
- [ ] I've two LGTMs/Approvals.
- [ ] I've fixed or replied to all my code-review comments.
- [ ] I've manually tested with a buddy.
- [ ] I've squashed my commits into one.

### CODE REVIEW

File: src/components/third-party/Popover/styles.scss [hunk 1]
```diff
@@ -174,7 +174,6 @@ $arrow-border-size: 1px;
   &.popover-light {
     $theme-background-color: $color-white;
     $theme-border-color: $color-gray-lighter;
-    @include theme-popover($theme-background-color, $theme-border-color);
```
Comment (veegandhi) [inline src/components/third-party/Popover/styles.scss]: avoid re-applying default arrow styles

File: src/components/third-party/Popover/styles.scss [hunk 2]
```diff
@@ -44,7 +44,7 @@ $arrow-border-size: 1px;
 
   .aui--popover-arrow {
     position: absolute;
-    border: 0 solid $color-border-light; // the color of the outer triangle
+    border: 0 solid $color-border-lighter; // the color of the outer triangle
```
Comment (veegandhi) [inline src/components/third-party/Popover/styles.scss]: Applying correct color as default

File: src/components/adslot-ui/AlertInput/styles.scss [hunk 3]
```diff
@@ -76,7 +76,7 @@
       border-color: $color-positive;
       color: $color-positive;
 
-      > .arrow {
+      > .aui--popover-arrow {
```
Comment (nimishjha) [inline src/components/adslot-ui/AlertInput/styles.scss]: Shouldn't this be `.aui__popover-arrow`?

Comment (veegandhi) [reply src/components/adslot-ui/AlertInput/styles.scss]: From memory adslot-ui has a sass lint rule for no underscore class names. So we chose to go with hyphens as the goal was to achieve a namespace.

**Comment by lightbringer1991** [comment]
merging...

**Pull request merged**